### PR TITLE
Feature/data download

### LINKED ISSUE #707: Data download
Add a new page where the users can download the data.

1. Bulk Download becomes Data download that is a link to the new page (see attached mockup)

**css**
```
.dataDownload {
    width: 100%;
    display: grid;
    grid-template-columns: 1fr auto auto;
    align-items: center;
    grid-column-gap: 16px;
    grid-row-gap: 16px;
}

.dataDownload__sep {
    grid-column: 1/4;
    border-top: 1px solid #e2e2e2;
}
```

**Mockup** 
Screenshot 2021-02-02 at 09 25 02

**Data**
[FRA2020AnalysisTables.zip](https://github.com/openforis/fra-platform/files/5909941/FRA2020AnalysisTables.zip)

### PR DESCRIPTION
**Pull request by minotogna**
Resolves #707

### CODE REVIEW

File: webapp/app/assessment/components/section/assessmentSectionView/assessmentSectionView.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ import { fetchLastSectionUpdateTimestamp, resetSectionUpdateTimestamp } from '@w
 import AssessmentSection from './assessmentSection'
 import useSectionTables from './useSectionTables'
 
+import { documentScrollTo } from '../../../../../utils/domUtils'
```
Comment (sorja) [inline webapp/app/assessment/components/section/assessmentSectionView/assessmentSectionView.js:19]: can we import with @?

File: webapp/pages/Assessment/DataDownload/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import './dataDownload.less'
+import React, { useEffect } from 'react'
+
+import { useI18n } from '../../../components/hooks'
+import { documentScrollTo } from '../../../utils/domUtils'
+
+import Icon from '../../../components/icon'
```
Comment (sorja) [inline webapp/pages/Assessment/DataDownload/index.js:7]: @webapp/components?

File: webapp/app/assessment/components/section/assessmentSectionView/assessmentSectionView.js (hunk 1 above)
Comment (minotogna) [reply webapp/app/assessment/components/section/assessmentSectionView/assessmentSectionView.js:19]: my IDE is not recognising them anymore. I would prefer we switch to normal import

**Review by sorja — CHANGES_REQUESTED** [review]
Change imports to use aliases

File: webapp/pages/Assessment/DataDownload/index.js (hunk 2 above)
Comment (minotogna) [reply webapp/pages/Assessment/DataDownload/index.js:7]: see above

**Review by sorja — APPROVED** [review]

**Review by sorja — APPROVED** [review]

**Pull request merged**